src/main/java/oxgame/app/RoundBuffer.java

```diff
@@ -17,10 +17,9 @@ public void addPlayer(Player player) throws UnsupportedOperationException{
 
```

Comment:
UnsupportedOperationException is a runtime exception change it to java.lang.Exception

Reply:
Ready to commit.